core/types/block_ext_test.go

```diff
@@ -174,7 +173,7 @@ func TestBlockWithNonZeroFields(t *testing.T) {
 // bodyWithNonZeroFields returns a [Body] and a [BlockBodyExtra],
 // each with all fields set to non-zero values.
 // The [BlockBodyExtra] extra payload is set in the [Body] via `extras.Block.Set`
-// and the extra copying done in [ethtypes.Block.Body].
+// and the extra copying done in [Block.Body].
```

Comment:
godoc does not support accessing a field from an aliased type (at least my IDE doesn't), maybe just change to

```suggestion
// and the extra copying done in `Block.Body()`.
```